### PR TITLE
Update CHANGELOG for 2.30.0

### PR DESCRIPTION
**Pull request by qiwzhang**
Signed-off-by: Wayne Zhang <qiwzhang@google.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Release 2.30.0 03-08-2021
+
+- Update service_control_client_cxx with improved quota cache. (#573)
+- b/194945474: Unify route match policy with CORS filter (#575)
```
Comment (nareddyt) [inline CHANGELOG.md:4]: remove bug number

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+- Issue 533 update envoy 16319 jwks retries flags (#564)
```
Comment (nareddyt) [inline CHANGELOG.md:5]: Remove issue number

**Review by nareddyt — APPROVED** [review]

**Review by TAOXUY — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (qiwzhang) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (qiwzhang) [reply CHANGELOG.md]: Done

**Pull request merged**